version.py

```diff
@@ -1,2 +1,2 @@
 ZULIP_VERSION = "1.6.0+git"
-PROVISION_VERSION = '5.7'
+PROVISION_VERSION = '5.8'
```

Comment:
this should probably go to 6.0, because downgrading will require a provision, too.